scheduler/backend/backend.go

```diff
@@ -51,7 +51,7 @@ func (b Backends) AvailabilityZone(backendID string) (string, error) {
 			return backend.AvailabilityZone, nil
 		}
 	}
-	return "", errors.New(fmt.Sprintf("No backend with ID %s found", backendID))
+	return "", errors.New(fmt.Sprintf("No backend with GUID %s found", backendID))
```

Comment:
Can we be consistent with naming. If we want to go with GUID as the reference to a backend can we also rename the variables to `backendGUID` or even better `cellGUID` since we decided a 'cell' is the thing where the cluster 'nodes' live on.